.dotnet.azure/sdk/openai/Azure.AI.OpenAI/src/Custom/Chat/AzureChatExtensions.cs

```diff
@@ -0,0 +1,89 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.AI.OpenAI.Internal;
+using System.Diagnostics.CodeAnalysis;
+
+#pragma warning disable AZC0112
+
+namespace Azure.AI.OpenAI.Chat;
+
+public static partial class AzureChatExtensions
```

Comment:
Do we need to add Experimental to the class too?

Reply:
If so, the other "Extensions" like `AzureImageExtensions`  will need it too.

Reply:
Likely doesn't hurt. All the methods have it and it's purely extension methods, but more `[Experimental`], fewer problems!